### PR TITLE
New NRT Rules Created

### PR DESCRIPTION
**Pull request by petebryan**
Change(s):
   Created several new NRT rule templates:
   - Created from existing scheduled detections
   - Some minor changes to support NRT limitations.

1.   AWS Login without MFA
2.   ADFS Trust Modifications
3.   New App or SP Credential
4.   PIM request rejected
5.   User added to privileged group
6.   New ADFS Server
7.   KeyVault Sensitive Operation
8.   Mining Pool Detection (2 detections)
9.   Malicious inbox rules (2 detections)
10. Security Logs being cleared
11. B64 Encoded PE files or command lines (2 detections)
12. MFA rejected by user

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes

_Note: If updating a detection, you must update the version field._

> After the submission has been made, please look at the Validation Checks.

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Detections/AWSCloudTrail/NRT_AWS_ConsoleLogonWithoutMFA.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+id: 0ee2aafb-4500-4e36-bcb1-e90eec2f0b9b
+name: NRT Login to AWS Management Console without MFA
+description: |
+  'Multi-Factor Authentication (MFA) helps you to prevent credential compromise. This alert identifies logins to the AWS Management Console without MFA.
+  You can limit this detection to trigger for adminsitrative accounts if you do not have MFA enabled on all accounts.
```
Comment (aprakash13) [inline Detections/AWSCloudTrail/NRT_AWS_ConsoleLogonWithoutMFA.yaml:5]: Small typo with the word Administrative

File: Detections/SigninLogs/NRT_MFARejectedbyUser.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+id: 3617d76d-b15e-4c6f-985e-a1dac73c592d
+name: NRT MFA Rejected by User
+description: |
+  'Identifies accurances where a user has rejected an MFA prompt. This could be an indicator that a threat actor has compromised the username and password of this user account and is using it to try and log into the account.
```
Comment (aprakash13) [inline Detections/SigninLogs/NRT_MFARejectedbyUser.yaml:4]: probably a small typo in the word occurrences

**Review by aprakash13 — CHANGES_REQUESTED** [review]
The queries look great. Thanks @petebryan.  A couple of small typos that we can probably fix and this would be ready to be merged

File: Detections/SigninLogs/NRT_MFARejectedbyUser.yaml (hunk 2 above)
Comment (petebryan) [reply Detections/SigninLogs/NRT_MFARejectedbyUser.yaml]: Fixed

**Comment by petebryan** [comment]
Thanks so much @aprakash13! Good spot on those typos, all updated.

**Review by aprakash13 — APPROVED** [review]
Thanks Pete.

**Pull request merged**